fhir-helper/src/jvmTest/kotlin/care/data4life/sdk/helpers/r4/MedicationRequestHelperTest.kt

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2020 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"), 
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of 
+ * applications/third-party applications shall require the conclusion of a license agreement 
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party 
+ * applications and/or if you’d like to contribute to the development of the SDK, please 
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.helpers.r4
+
+import com.google.common.truth.Truth.assertThat
+import care.data4life.fhir.r4.model.*
+import care.data4life.sdk.util.StringUtils
+import io.mockk.every
+import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
+import org.junit.Assert.fail
+import org.junit.Before
+import org.junit.After
+import org.junit.Test
+import java.util.*
+import io.mockk.unmockkAll
+
+class MedicationRequestHelperTest {
+
+    private var patient: Patient? = null
+    private var medication: Medication? = null
+    private var dosage: Dosage? = null
+
+    @Before
+    fun setup() {
+        patient = Patient()
+        medication = Medication()
+        dosage = Dosage()
+    }
+
+    @After
+    fun tearDown() {
+        unmockkAll()
+    }
+    @Test
+    fun buildWith_shouldReturnMedRequest_whenAllArgsProvided() {
+        // Given
+        val MEDICATION_ID = "medicationId"
+        val PATIENT_ID = "patientId"
+
+        mockkStatic(StringUtils::class)
```

Comment:
this has to be `mockkObject(StringUtils)` as the StringUtils is not a static Java file but an Kotlin object